securedrop_client/gui/source/delete/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+Everything necessary for a journalist to delete a source.
+
+Copyright (C) 2021  The Freedom of the Press Foundation.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+# Import classes here to make possible to import them from securedrop_client.gui.source.delete
+from securedrop_client.gui.source.delete.dialog import DeleteSourceDialog  # noqa: F401
```

Comment:
Okay, so you are hiding the `dialog` module from the path so that you can `from securedrop_client.gui.source.delete import DeleteSourceDialog` when you actually need to import `DeleteSourceDialog` vs `from securedrop_client.gui.source.delete.dialog import DeleteSourceDialog`. Either way works for me, but I tend to try to keep my `__init__.py` empty and kind of prefer the simplicity and explicit-y when importing classes, functions, and variables from somewhere.